update-since-todo.sh

```diff
@@ -28,6 +30,6 @@ do
         sed -i.bak "$sedExpr" "$file"
         rm -f "$file.bak"
     else
-        echo -e "\tNot updating file, no tag found. Normal if the associated PR/commit is not merged and released yet"
+        echo -e "\tNot updating file, no tag found. Normal if the associated PR/commit is not merged and released yet; otherwise make sure to fetch tags from jenkinsci/jenkins"
```

Comment:
Would it be worth the extra cost to insert a `git fetch --tags` once at the start of the script?

Reply:
Would rather not have this script modify the clone.